ddtrace/tracer/span.go

```diff
@@ -369,9 +369,16 @@ func (s *span) finish(finishTime int64) {
 	s.context.finish()
 }
 
+func normalizeSpan(s *span) {
+	if s.Duration < 0 {
+		s.Duration = 0
+	}
+}
+
 // newAggregableSpan creates a new summary for the span s, within an application
 // version version.
 func newAggregableSpan(s *span, cfg *config) *aggregableSpan {
+	normalizeSpan(s)
```

Comment:
This doesn't seem like the right place for span normalization.

Reply:
In that case, we apply normalization on both traces and stats. If that's OK, I can move it. What do you think?

Reply:
I don't see why not. A negative duration is invalid in both... Plus, they shouldn't have different field values if they represent the same exact thing...

Reply:
sounds good, just moved it.